crates/bevy_sprite/src/render/mod.rs

```diff
@@ -186,6 +187,20 @@ pub struct ExtractedSprite {
     pub flip_y: bool,
 }
 
+#[derive(Clone, Copy)]
+pub struct Transform2d {
```

Comment:
I feel quite strongly that this should be used in the public APIs for both UI and sprites, along with a public z-layer component. I'm not sure if this PR is the place for it though.

Reply:
This should be done is it's own PR to not go out of scope for this one.

Reply:
I renamed the struct to make it clear it is not to be used as a general `Transform2d`